llvm/lib/DebugInfo/GSYM/FunctionInfo.cpp

```diff
@@ -32,6 +33,8 @@ raw_ostream &llvm::gsym::operator<<(raw_ostream &OS, const FunctionInfo &FI) {
     OS << FI.OptLineTable << '\n';
   if (FI.Inline)
     OS << FI.Inline << '\n';
+  if (FI.CallSites)
+    OS << *FI.CallSites << '\n';
```

Comment:
`CallSites` is an optional value like `Inline` above. I wonder why this pointer pattern is different. Perhaps `Inline` is a bug? Do we actually have a test coverage on this code?

Reply:
I don't think this code is used at all. I added it here for consistency - but I could not find any usage / test coverage.